boot/bootutil/zephyr/CMakeLists.txt

```diff
@@ -34,7 +34,13 @@ target_include_directories(MCUBOOT_BOOTUTIL INTERFACE
 )
 endif()
 
-if(CONFIG_BOOT_USE_MBEDTLS)
+if(CONFIG_BOOT_USE_PSA_CRYPTO)
+target_include_directories(MCUBOOT_BOOTUTIL INTERFACE
+   ${ZEPHYR_MBEDTLS_MODULE_DIR}/include
+)
```

Comment:
2 space indent for cmake. Is this going to be submitted upstream?

Reply:
Having a look below, wouldn't be keeping the existing one and adding a new one for psa crypto work better since you're not changing a piece of the upstream code?

Reply:
The general idea is to, at some point, bring this upstream.
I am keeping the PSA code separate because the change is significant and the code is already badly ifdefed in the verification/encryption calls.

Reply:
Not resolved